cmd/broker/filter/main.go

```diff
@@ -67,6 +84,30 @@ func main() {
 		logger.Fatal("Unable to start the receiver", zap.Error(err), zap.Any("receiver", receiver))
 	}
 
+	// Metrics
+	e, err := prometheus.NewExporter(prometheus.Options{Namespace: metricsNamespace})
+	if err != nil {
+		logger.Fatal("Unable to create Prometheus exporter", zap.Error(err))
+	}
+	view.RegisterExporter(e)
+	sm := http.NewServeMux()
+	sm.Handle("/metrics", e)
+	metricsSrv := &http.Server{
+		Addr:         fmt.Sprintf(":%d", metricsPort),
+		Handler:      e,
+		ErrorLog:     zap.NewStdLog(logger),
+		WriteTimeout: writeTimeout,
+	}
+
+	err = mgr.Add(&utils.RunnableServer{
```

Comment:
Why are you using the manager for this?

Reply:
. Quickest way to get this in for 0.6. The Broker receiver still uses the Manager, and the Broker ingress already uses the Manager to do metrics exporting, making this a convenient copy and paste.

Reply:
Since getting this into 0.6 is no longer a forcing function, could we either as part of this work, or as a followup with a tracking issue move these to using the pkg/controller. Afaik these are the only things still using the manager.

Reply:
0.7 is now the forcing function :) I created #1378 to track.